### DISCUSSION TITLE
Tailwind CSS not loading when deployed to netlify

### DISCUSSION THREAD

**Discussion by foolhardys** [tailwindlabs/tailwindcss#12165] [Help]
I have  a project build with tailwind css. The project is working fine in my development server but on deploying the Tailwind CSS is not loading. Please help.

[https://github.com/foolhardys/Araayna-Saxena](url) Github Repo Link
[https://araaynaportfolio.netlify.app/](url) Deploy link

**Comment by wongjn** [comment]
Getting *Site Not Found* for https://araaynaportfolio.netlify.app/. Did you mean https://araayna-saxena.vercel.app/? It seems to be loading fine.
